allms/domain/response.py

```diff
@@ -5,13 +5,18 @@
 from allms.domain.input_data import InputData
 
 
+class ResponseWithError(BaseModel):
```

Comment:
The name of the class is confusing and it suggests the response will have an error, whereas it's used to store the response with extracted JSON. I'd call it `JSONResponse` or `JOSNParsedResponse`

Reply:
> The name of the class is confusing and it suggests the response will have an error, whereas it's used to store the response with extracted JSON. I'd call it `JSONResponse` or `JOSNParsedResponse`

Name changed to `ResponseParsingOutput`.